spec/features/user_can_view_past_rentals_spec.rb

```diff
@@ -21,7 +16,7 @@
 
       expect(page).to have_content(rental_one.id)
       expect(page).to have_content('November 11, 2015')
-      expect(page).to have_content('$89')
+      expect(page).to have_content('$190')
```

Comment:
Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.